client/core/trade.go

```diff
@@ -744,6 +591,11 @@ func (t *trackedTrade) recalcFilled() (matchFilled, canceled uint64) {
 	if t.cancel != nil && t.cancel.matches.maker != nil {
 		canceled = t.cancel.matches.maker.Quantity
 	}
+	return
+}
+
+func (t *trackedTrade) recalcFilled() (matchFilled, canceled uint64) {
+	matchFilled, canceled = t.filled()
```

Comment:
Possible refactor to extract from this PR. (get `matchFilled` amount without calling `SetFill`)